src/pokemon_data.c

```diff
@@ -931,3 +931,422 @@ const s8 gNatureStatTable[][5] = {
+    {0xcff37e37, 0x00b41ef6}, // 0b0010110100000111101111011011001111111100110111111000110111 (Aggron)
+    {0xa5b33664, 0x00403e36}, // 0b0001000000001111100011011010100101101100110011011001100100 (Castform)
+    {0xe5b78625, 0x00403e82}, // 0b0001000000001111101000001011100101101101111000011000100101 (Volbeat)
+    {0xe5b78625, 0x00403e82}, // 0b0001000000001111101000001011100101101101111000011000100101 (Illumise)
+    {0x84350720, 0x00001e18}, // 0b0000000000000111100001100010000100001101010000011100100000 (Lileep)
+    {0x86354720, 0x00a01e58}, // 0b0010100000000111100101100010000110001101010100011100100000 (Cradily)
+    {0xcc110624, 0x00841ed0}, // 0b0010000100000111101101000011001100000100010000011000100100 (Anorith)
+    {0xce514624, 0x00a41ed0}, // 0b0010100100000111101101000011001110010100010100011000100100 (Armaldo)
+    {0xb49b8e28, 0x0041bf03}, // 0b0001000001101111110000001110110100100110111000111000101000 (Ralts)
+    {0xb49b8e28, 0x0041bf03}, // 0b0001000001101111110000001110110100100110111000111000101000 (Kirlia)
+    {0xb49bce28, 0x0041bf03}, // 0b0001000001101111110000001110110100100110111100111000101000 (Gardevoir)
+    {0xc4130632, 0x00a41ee4}, // 0b0010100100000111101110010011000100000100110000011000110010 (Bagon)
+    {0xc4130632, 0x00a41ee4}, // 0b0010100100000111101110010011000100000100110000011000110010 (Shelgon)
+    {0xc6534632, 0x00ac5ee4}, // 0b0010101100010111101110010011000110010100110100011000110010 (Salamence)
+    {0x00000000, 0x00000000}, // 0b0000000000000000000000000000000000000000000000000000000000 (Beldum)
+    {0xf613c620, 0x00e40ed9}, // 0b0011100100000011101101100111110110000100111100011000100000 (Metang)
+    {0xf613c620, 0x00e40ed9}, // 0b0011100100000011101101100111110110000100111100011000100000 (Metagross)
+    {0xcf994621, 0x00a00e52}, // 0b0010100000000011100101001011001111100110010100011000100001 (Regirock)
+    {0xc79b7261, 0x00a00e02}, // 0b0010100000000011100000001011000111100110110111001001100001 (Regice)
+    {0xc79b4621, 0x00a00ed2}, // 0b0010100000000011101101001011000111100110110100011000100001 (Registeel)
+    {0xc79b727c, 0x03b00e42}, // 0b1110110000000011100100001011000111100110110111001001111100 (Kyogre)
+    {0xcff946b2, 0x00a60ef6}, // 0b0010100110000011101111011011001111111110010100011010110010 (Groudon)
+    {0xc7f376b6, 0x03ba0eb6}, // 0b1110111010000011101011011011000111111100110111011010110110 (Rayquaza)
+    {0xb7bbd63e, 0x035c5e93}, // 0b1101011100010111101001001110110111101110111101011000111110 (Latias)
+    {0xb7bbd63e, 0x035c5e93}, // 0b1101011100010111101001001110110111101110111101011000111110 (Latios)
+    {0xb59bc62c, 0x00408e93}, // 0b0001000000100011101001001110110101100110111100011000101100 (Jirachi)
+    {0xf5bbde2d, 0x00e58fc3}, // 0b0011100101100011111100001111110101101110111101111000101101 (Deoxys)
+    {0xb41b8e28, 0x00419f03}, // 0b0001000001100111110000001110110100000110111000111000101000 (Chimecho)
+};
```

Comment:
If you changed the move, you would have to set the TM01 flag for all Pokemon, which is error prone.

```c
struct TMLearn {
    u64 FocusPunch:1;
}
const struct TMLearn gTMLearns = {
    /* Machamp */ {
        .FocusPunch=1,
    }
};
```

If you did this:
```diff
 struct TMLearn {
-    u64 FocusPunch:1;
+    u64 HyperBeam:1;
 }
```

You would get compile errors on Pokemon that learn Focus Punch, and you wouldn't have to worry about Pokemon that don't. You also don't run the risk of any Pokemon learning Hyper Beam when they shouldn't. Also, you don't have to explicitly say that a Pokemon does not learn a move (but you can if you like). This behavior is very desirable.